about/overview.md

```diff
@@ -1,20 +1,9 @@
 (about-the-project)=
-# About the Managed JupyterHub Service
+# Overview
 
 2i2c JupyterHubs offer data science environments to multiple users via a JupyterHub that runs in the cloud.
```

Comment:
I expect some of our customers will not identify as belonging to "data science". I suggest we say "2i2c JupyterHubs offer interactive computing environments designed to serve multiple users via a JupyterHub that runs in the cloud."